package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pretest": "jshint .",
-    "test": "mocha -R spec"
+    "test": "mocha -R spec --timeout 60000"
```

Comment:
The timeout to run only one test to verify the install. The timeout is just for safety. If `npm i` finishes earlier, the build moves on.